src/components/ScheduledMatch.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+
+export default class ScheduledMatch extends Component {
+
+  render() {
+    return (
+      <div className="card mt-4">
+        <div className="card-body">
+          <div className="container col-xs-4">
+            <div className="row">
+              <div className="col-lg-4 col-sm-4 col-xs-12 float-left">
+                <div className="card border-0">
+                  <div className="card-body">
+                    <h4 className={"text-center " + (this.props.homeScore > this.props.awayScore ? "text-success" : "text-secondary")}>{this.props.home}
+                      <div><small>{this.props.winRatioHomeTeam}</small></div>
+                    </h4>
+                  </div>
+                </div>
+              </div>
+              <div className=" col-lg-4 col-sm-4 col-xs-12">
+                <div className="card border-0">
+                  <div className="card-body">
+                    <h4 className="text-center">
+                      <span className={(this.props.homeScore > this.props.awayScore ? "text-success" : "text-secondary")}>{this.props.homeScore}</span>
+                      {'\u00A0'}:{'\u00A0'}
```

Comment:
whats this unicode ? use &nbsp; ... and you dont need react templating for rendering constants. just put there &nbsp;:&nbsp;